packages/react-storage/src/components/StorageBrowser/composables/DataTable/dataCells/TextDataCell.tsx

```diff
@@ -22,11 +23,16 @@ export const TextDataCell = ({
     <ViewElement className={`${CLASS_BASE}__table-text-data-cell`}>
       {icon && (
         <IconElement
-          className={`${CLASS_BASE}__table-text-data-cell-icon--${icon}`}
+          className={`${CLASS_BASE}__table-text-data-cell-icon ${CLASS_BASE}__table-text-data-cell-icon--${icon}`}
```

Comment:
Why do we need the additional `className`?

Reply:
Allows us to target all the icons here: https://github.com/aws-amplify/amplify-ui/pull/5960/files/98b919a4b810777a1e2d5d29c1476efd50b411ac#diff-682958b901d15d63290647a5be96b94752c782bbd462f4db33d749299c3243adR358-R360. We could use a wildcard, but I feel like this is cleaner.